tests/lib/gendeveloper1model/main.go

```diff
@@ -48,6 +49,11 @@ import (
 )
 
 func main() {
+	if len(os.Args) > 1 && os.Args[1] == "show-key" {
```

Comment:
maybe renaming the binary to say `gendeveloper1` with commands `sign-model` and `show-key` would be better in the long run. This would need an update to the readme in tests/lib/assertions too.

Reply:
Done.